packages/fluentui/docs/src/examples/components/Form/Usage/index.tsx

```diff
@@ -4,6 +4,11 @@ import ExampleSection from '../../../../components/ComponentDoc/ExampleSection';
 
 const Usage = () => (
   <ExampleSection title="Usage">
+    <ComponentExample
+      title="Form Components"
```

Comment:
I suggest it to move to a separate section.

Reply:
What would be the section name?

Reply:
What about Variants? :)

Reply:
moved